Public/Src/Engine/Scheduler/Fingerprints/PipFingerprinter.cs

```diff
@@ -336,6 +344,11 @@ protected virtual void AddWeakFingerprint(IFingerprinter fingerprinter, Process
             }
 
             fingerprinter.AddCollection<int, ReadOnlyArray<int>>("SuccessExitCodes", process.SuccessExitCodes, (h, i) => h.Add(i));
+
+            if (process.ChangeAffectedInputListWrittenFilePath.IsValid)

```

Comment:
Why is the path `ChangeAffectedInputListWrittenFilePath` is not part of the fingerprint? What if I change that path?

Reply:
This is an internal used file. We never defined it as a input or output file. Thurs, its location shouldn't matters and shouldn't be in the fingerprint.

Reply:
However, you allow users to set its location, don't you? If that's the case, I think the path should go to the fingerprint as well.

---
In reply to: [336684057](https://github.com/microsoft/BuildXL/pull/1064#discussion_r336684057) [](ancestors = 336684057)